python/hsfs/core/vector_server.py

```diff
@@ -75,35 +109,59 @@ def __init__(
         self._required_serving_keys = []
 
         self._transformation_function_engine = (
-            transformation_function_engine.TransformationFunctionEngine(
-                feature_store_id
-            )
+            tf_engine_mod.TransformationFunctionEngine(feature_store_id)
         )
         self._transformation_functions = None
-        self._online_store_sql_client: Optional[
-            online_store_sql_client.OnlineStoreSqlClient
-        ] = None
+        self._online_store_sql_client = None
+
+        self._online_store_rest_client_engine = None
+        self._init_online_store_rest_client: Optional[bool] = None
+        self._init_online_store_sql_client: Optional[bool] = None
+        self._default_online_store_client = None
+
+        self.set_return_feature_value_handlers(features=self._features)
 
     def init_serving(
         self,
         entity: Union[feature_view.FeatureView, training_dataset.TrainingDataset],
         external: Optional[bool] = None,
         inference_helper_columns: bool = False,
         options: Optional[Dict[str, Any]] = None,
+        init_online_store_sql_client: Optional[bool] = None,
```

Comment:
Shouldn't we ideally set one of init_online_store_sql_client or init_online_store_rest_client as True by default so as to make it more easier for users.